frontend/src/components/projectshow.css

```diff
@@ -47,3 +47,9 @@ button:hover {
 tfoot button {
     margin-top: 1em;
 }
+.tableButton{
+
+
+    width: 7%;
```

Comment:
eventuell lieber eine feste Breite wählen? oder in min-width ändern, damit der inneliegende Text beim Schmaler schieben nicht breiter wird als sein Parent?